spec/factories/user.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+
+    factory :user do
+      email    'alice@example.com'
+      username 'Alicem'
+      password '12345678'
+      password_confirmation '12345678'
+
+      factory :failed_user do
+      password_confirmation '1234556789'
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 12, 2 is not aligned with `factory :user do` at 3, 4